MaxText/layers.py

```diff
@@ -123,8 +125,15 @@ def dot_product_attention(query: Array,
   query = LayerNorm(dtype=dtype, name='query_layer_norm', kernel_axes = ('heads',))(query)
   key = LayerNorm(dtype=dtype, name='key_layer_norm', kernel_axes = ('heads',))(key)
 
-  # `attn_weights`: [batch, num_heads, q_length, kv_length]
-  attn_weights = jnp.einsum('bqhd,bkhd->bhqk', query, key)
+  # QK Product, a.k.a `attn_weights`: [batch, num_heads, q_length, kv_length]
+  if not cfg.int8_training:
+    attn_weights = jnp.einsum('bqhd,bkhd->bhqk', query, key)
+  else: 
+    aqt_cfg = maxtext_sweeps.sweep1(cfg.fwd_int8_qk, cfg.dlhs_int8_qk, cfg.drhs_int8_qk, use_fwd_quant=cfg.aqt_use_fwd_quant)
```

Comment:
Why are you not passing 
      use_dummy_static_bound=cfg.aqt_use_dummy_static_bound,
      rng_type=cfg.aqt_rng_type,

Reply:
Great catch! Fixed in both places it was missing.